mediaire_toolbox/transaction_db/migrations.py

```diff
@@ -90,6 +90,21 @@
         "  FOREIGN KEY (user_id) REFERENCES users(id),"
         "  FOREIGN KEY (site_id) REFERENCES sites(id)"
         ");"
+    ],
+    19: [
+        "ALTER TABLE transactions ADD COLUMN patient_consent_date DATETIME DEFAULT NULL;",
+        ("UPDATE transactions SET"
+         "  patient_consent_date = creation_date "
+         "WHERE"
+         "  patient_consent = 1"
+         "  AND data_uploaded IS NOT NULL"
```

Comment:
I think you want to say here creation_date is NOT NULL, no?

Reply:
good catch :see_no_evil: